### DISCUSSION TITLE
Unclear animations step in ROADMAP.md

### DISCUSSION THREAD

**Discussion by badcodecat** [hecrj/iced#1741] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Is this issue related to iced?

- [X] My hardware is compatible and my graphics drivers are up-to-date.

### What happened?

ROADMAP.md says that Animations(#31 ) is not yet complete. However it appears that animations were added in 0.7.0 and it looks like the associated issues are closed.

### What is the expected behavior?

I was expecting to see this issue collapsed and added to the list of completed steps. If it is not yet complete I would have liked to have seen an associated open issue.

### Version

master

### Operative System

Linux

**Comment by 13r0ck** [comment]
Checking off the roadmap item may be appropriate already. But there are a few more cases where animation logic isn't handled as good as it could be. https://github.com/pop-os/cosmic-time attempts to tackle that
